src/test/java/driver/ChromeSingleton.java

```diff
@@ -0,0 +1,26 @@
+package driver;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+public class ChromeSingleton {
+    private static ChromeSingleton instance;
```

Comment:
very strange singleton implementation. refactor it. and please do not use "static" everywhere if you are not 1000% you need it

Reply:
fixed.